### PR TITLE
CI Cleanup.

### PR DESCRIPTION
**Pull request by jctanner**
#### What is this PR doing:
* Eliminate unnecessary spinup of the UI
* Move common RUN_INTEGRATION to standalone folder
* Add gettext in dev containers for translation builds
* Disable pulp telemetry in CI 

#### Reviewers must know:

**PR Author & Reviewers**: Keep or remove backport labels per [Backporting Guidelines](https://github.com/ansible/galaxy_ng/wiki/Backporting-Guidelines)
**Reviewers**: Look for sound code, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/), docs & test coverage
**Merger**: When merging, include the Jira issue link in the squashed commit

### CODE REVIEW

**Comment by alikins** [comment]
Could use a little more detail about in the commit messages.  It's not clear what is being cleaned up or why at first glance.

File: .github/workflows/ci_insights.yml [hunk 1]
```diff
@@ -58,13 +52,7 @@ jobs:
...
 
       - name: build stack
-        run: make docker/build
-
-      - name: run migrations
-        run: make docker/migrate
-
-      - name: load test data
-        run: make docker/loaddata
+        run: make docker/all
```
Comment (alikins) [inline .github/workflows/ci_insights.yml:55]: I kind of like the idea of these steps being separate in the process (and more so in the action runs output). 

Is there an advantage to smashing it into one?  
Is it to avoid duplicating 'make docker/all' logic in multiple places?

File: dev/Dockerfile.base [hunk 2]
```diff
@@ -32,6 +32,7 @@ RUN set -ex; \
 RUN set -ex; \
     touch /var/lib/rpm/* \
     && dnf -y install \
+        gettext \
```
Comment (alikins) [inline dev/Dockerfile.base:35]: What changed that makes gettext optional now?

File: dev/standalone/RUN_INTEGRATION.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
Comment (alikins) [inline dev/standalone/RUN_INTEGRATION.sh:1]: Could/should this be a separate pr?

File: .github/workflows/ci_insights.yml (hunk 1 above)
Comment (jctanner) [reply .github/workflows/ci_insights.yml:55]: Because new developers already have too many things to learn without also having to remember what order to run the make targets that get the stack ready.

File: dev/standalone/RUN_INTEGRATION.sh (hunk 3 above)
Comment (jctanner) [reply dev/standalone/RUN_INTEGRATION.sh:1]: dev/common/RUN_INTEGRATION.sh is going away and this continues the trend of having a script per compose profile.

File: dev/Dockerfile.base (hunk 2 above)
Comment (jctanner) [reply dev/Dockerfile.base:35]: It's used when building django translations.

**Comment by bmbouter** [comment]
Thank you for disabling telemetry! We only want end user systems to have it enabled.

**Review by awcrosby — APPROVED** [review]
